vividus-plugin-mongodb/src/main/resources/spring.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
+       default-lazy-init="true">
+
+    <bean id="mongoDbSteps" class="org.vividus.bdd.steps.mongodb.MongoDbSteps">
```

Comment:
https://docs.spring.io/spring-data/mongodb/docs/current/reference/html/#mongodb-connectors
Does it make sense to reuse spring provided functionality?

Reply:
spring mongodb support is built on mongo native driver, I don't see any advantage of using wrappers and pulling additional dependencies in bounds of our needs